src/main/java/org/apache/ibatis/cache/impl/PerpetualCache.java

```diff
@@ -28,7 +28,7 @@
 
   private final String id;
 
-  private Map<Object, Object> cache = new HashMap<>();
+  private final Map<Object, Object> cache = new HashMap<>(8);
```

Comment:
Why change the `initialCapacity` from default?  You should be explain a reason for changing it.

Reply:
May be ，It's better to use the default value in this place，I'll fix it。